modules/schematics/src/ng-add/schema.json

```
@@ -0,0 +1,16 @@
+{
+  "$schema": "http://json-schema.org/schema",
+  "id": "SchematicsNgRxRoot",
```

Might be a bit weird, but the other packages are using the following convention:

```suggestion
  "id": "SchematicsNgRxSchematics",
```